Specs/Number/Korean/NumberModel.json

```diff
@@ -422,7 +422,7 @@
   {
     "Input": " 123456789101231",
     "Comment": "PendingValidation",
-    "NotSupportedByDesign": "javascript,python,java",
+    "NotSupportedByDesign": "dotNet,javascript,python,java",
```

Comment:
Why add dotnet to NotSupportedByDesign?

Reply:
@neudurgeshp

Reply:
It was by mistake. Re-enabling that.

Reply:
Re-enabled that. Also removed the dotnet from NotSupportedByDesign from other cases as well and put into NotSupported if not supporting.